org.mozilla.firefox.BaseApp.json

```diff
@@ -96,6 +96,27 @@
                     "sha256": "d85195b9139d120cb6ba20199a19d3ef4aecdb7922eaead62994c704ea3da19e"
                 }
             ]
+        },
+        {
+            "name": "libspeechd",
+            "config-opts": [
+                    "--disable-python",
+                    "--with-systemdsystemunitdir=''",
+                    "--disable-rpath",
+                    "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/brailcom/speechd/releases/download/0.11.5/speech-dispatcher-0.11.5.tar.gz",
+                    "sha256": "1ce4759ffabbaf1aeb433a5ec0739be0676e9bdfbae9444a7b3be1b2af3ec12b"
+                }
+            ],
+            "cleanup": [
+                "/bin",
+                "/etc",
+                "/lib/speech-dispatcher"
```

Comment:
Add 

```
      - /libexec
      - /share/sounds/speech-dispatcher
      - /share/speech-dispatcher
```

too

and `/share/doc` to the top, pcsc is keeping a stray docs folder.

Reply:
done